test/options_test.cc

```diff
@@ -276,6 +277,141 @@ TEST_F(OptionsImplTest, RequestSource) {
   EXPECT_TRUE(util(*(options_from_proto.toCommandLineOptions()), *cmd));
 }
 
+// This test covers --RequestSourcePlugin, which can't be tested at the same time as --RequestSource
+// and some other options. This is the test for the inlineoptionslistplugin.
+TEST_F(OptionsImplTest, InLineOptionsListRequestSourcePlugin) {
```

Comment:
We try to name tests with the action you are testing/verifying.  It should be clear what a test is accomplishing before reading the code.

Reply:
That makes sense.